content/content-strategy/cms/purpose-cms/index.yml

```diff
@@ -0,0 +1,19 @@
+pagetitle: Purpose and place of a CMS
+description: Learn more about the specific place and purpose of a content management system in your content strategy. Choose the right CMS for your needs.
+weight: 80
+layout: layout/topic
+hidden: true
+theme: blue
+header:
+  - /_shared/globalheader.md
+  - /_shared/header.md
+main:
+  - intro.md
+  - heading.md 
+  - section.md #text
+  - diagram.md #diagram
+  - long-text-cms.md #longtext
+  - cards.md #approach
```

Comment:
Missing a `- /_shared/feedback.md` (feedback partial) at the end of the list, `main`

Reply:
Steve. Added